hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterWalManager.java

```diff
@@ -398,4 +414,33 @@ public void archiveMetaLog(final ServerName serverName) {
       LOG.warn("Failed archiving meta log for server " + serverName, ie);
     }
   }
+
+  private static Stoppable createDummyStoppable() {
+    return new Stoppable() {
+      private volatile boolean isStopped = false;
+
+      @Override
+      public void stop(String why) {
+        isStopped = true;
+      }
+
+      @Override
+      public boolean isStopped() {
+        return isStopped;
+      }
+    };
+  }
+
+  public ScheduledChore getOldWALsDirSizeUpdaterChore() {
```

Comment:
Can we have this Chore imlemented on its own class? That way we don't need to define new methods on MasterWalManager that just concerns this Chore logic, like the `createDummyStoppable` and `updateWalDirSize()` above.

Reply:
Sure, Will do that